samza-kafka/src/main/java/org/apache/samza/system/kafka/KafkaSystemConsumer.java

```diff
@@ -108,18 +107,15 @@ public KafkaSystemConsumer(Consumer<K, V> kafkaConsumer, String systemName, Conf
   /**
    * Create internal kafka consumer object, which will be used in the Proxy.
    * @param systemName system name for which we create the consumer
-   * @param clientId client id to use int the kafka client
-   * @param config config
-   * @return kafka consumer object
+   * @param kafkaConsumerConfig
+   * @return kafka consumer
    */
-  public static KafkaConsumer<byte[], byte[]> getKafkaConsumerImpl(String systemName, String clientId, Config config) {
+  public static KafkaConsumer<byte[], byte[]> getKafkaConsumerImpl(String systemName,
+      HashMap kafkaConsumerConfig) {
 
-    // extract kafka client configs
-    KafkaConsumerConfig consumerConfig = KafkaConsumerConfig.getKafkaSystemConsumerConfig(config, systemName, clientId);
+    LOG.info("KafkaClient properties for systemName {}: {}", systemName, kafkaConsumerConfig);
```

Comment:
Changed the log message.
I think the log-level concern is valid. But in case of issues with the Kafka consumption, this will be an important part of the debugging. I would suggest to leave it as info level for now and change it to debug when the code gets some millage.